README.md

```diff
@@ -21,10 +21,19 @@ Stack traces are preserved, by replacing removed types with white spaces.
 
 ```javascript
 const amaro = require('amaro');
-const { code } = amaro.transformSync("const foo: string = 'bar';");
+const { code } = amaro.transformSync("const foo: string = 'bar';", { mode: "strip-only" });
 console.log(code); // "const foo         = 'bar';"
 ```
 
+### Loader
+
+It is possible to user Amaro as a loader for TypeScript files.
+This allows the local Amaro version to override the Amaro version used by Node.js.
+
+```bash
+node --experimental-strip-types --import="amaro/register" script.ts
```

Comment:
Do you need `--experimental-strip-types` when you use the loader?

Reply:
See https://github.com/nodejs/amaro/pull/47#discussion_r1709203737. It is needed because the current implementation has no resolution.